client/cz-component-dash.html

```diff
@@ -19,7 +19,7 @@
         align-content: flex-start;
       }
 
-      .component {
+      .breakdown {
```

Comment:
again with this one. "breakdown" isn't really more generic than "component", and is more confusing. I'd actually like you to change this back before committing.